apps/examples/rtc/rtc_main.c

```diff
@@ -150,10 +161,230 @@ static void *rtc_test(void *arg)
 
 	close(fd);
 	sched_unlock();
-	printf("Rtc Accuracy Test Stopped well\n");
+	printf("RTC Test Stopped well\n");
 	return NULL;
 }
 
+#ifdef CONFIG_RTC_ALARM
+/****************************************************************************
+ * Name: alarm_handler
+ ****************************************************************************/
+
+static void alarm_handler(int signo, FAR siginfo_t *info, FAR void *ucontext)
+{
+	int ret;
+	int fd;
+	int secs;
+	int almndx;
+	FAR struct rtc_time alarm_recieve_time;
+
+	fd = open(RTC_DEVPATH, O_WRONLY);
+	if (fd < 0) {
+		fprintf(stderr, "ERROR: Failed to open %s: %d\n", RTC_DEVPATH, errno);
+		return;
+	}
+	almndx = info->si_value.sival_int;
+	if (almndx >= 0 && almndx < CONFIG_RTC_NALARMS) {
+		ret = ioctl(fd, RTC_RD_TIME, (unsigned long)&alarm_recieve_time);
+		if (ret < 0) {
+			fprintf(stderr, "ERROR: RTC_RD_TIME ioctl failed: %d\n", errno);
```

Comment:
Is there any reason using fprintf instead of printf?

Reply:
I thought of using it as gets to output via stderr stream, But anyways if we prefer printf, I will change it to printf and it will also print via stdout stream.